### PR TITLE
Prefers 5.6 if both 5.6 and 7.0 satisfies the req.

### LINKED ISSUE #52: Prefer PHP 5.6 over PHP 7.0
It seems like the memcached extension is not stable on PHP 7.0.

https://github.com/php-memcached-dev/php-memcached/issues/213

We should prefer PHP 5.6 until it's stable enough.

### PR DESCRIPTION
**Pull request by tmatsuo**
Fixes #52

### CODE REVIEW

File: scripts/build_images.sh [hunk 1]
```diff
@@ -19,4 +19,5 @@ docker build -t php-nginx php-nginx
 docker build -t php56 testapps/php56
 docker build -t php56_custom testapps/php56_custom
 docker build -t php70 testapps/php70
+docker build -t php56_70 testapps/php56_70
```
Comment (bshaffer) [inline scripts/build_images.sh]: This is a build with both PHP versions installed

**Comment by bshaffer** [comment]
LGTM, although this sure looks like a lot of images to build. Is that standard?

File: scripts/build_images.sh (hunk 1 above)
Comment (tmatsuo) [inline scripts/build_images.sh]: This is a virtual user app, which has a composer entry of php "^5.6|^7.0". For now we should choose PHP 5.6 for such apps.

**Comment by tmatsuo** [comment]
The images are for testing and I think it's fair.

**Comment by tmatsuo** [comment]
Thanks, merging!

**Pull request merged**